fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java

```diff
@@ -521,22 +525,29 @@ public List<List<String>> getInfos() {
                 GroupId groupId = entry.getValue();
                 info.add(groupId.toString());
                 info.add(entry.getKey());
-                StringBuffer sb = new StringBuffer();
+                StringBuilder tableIds = new StringBuilder();
+                StringBuilder tableNames = new StringBuilder();
                 for (Long tableId : group2Tables.get(groupId)) {
-                    if (sb.length() > 0) {
-                        sb.append(", ");
+                    if (tableIds.length() > 0) {
+                        tableIds.append(", ");
+                        tableNames.append(", ");
                     }
-                    sb.append(tableId);
-                    if (!validDbIdAndTableId(groupId.dbId, tableId)) {
-                        sb.append("*");
+                    tableIds.append(tableId);
+                    String tableName = getTableName(groupId.dbId, tableId);
+                    if (tableName == null) {
+                        tableIds.append("*");
```

Comment:
Why the tableIds use "*, tableNames use "NULL"?

Reply:
`*` means that this table is invalid(for example, in the recycle bin).